### PR TITLE
Personal/xiatia/new docker image options

### PR DESCRIPTION
**Pull request by ACMoretxj**
## Description

* Added the `latest` image tag while deploying.
* Use offline mode while validating using the official validator.

## Testing

All the unit tests are passed.

### CODE REVIEW

File: deploy/default-azuredeploy.json [hunk 1]
```diff
@@ -36,9 +36,10 @@
       "type": "string",
       "allowedValues": [
         "v1.0.0",
-        "v2.0.0"
+        "v2.0.0",
+        "latest"
       ],
-      "defaultValue": "v2.0.0"
+      "defaultValue": "latest"
```
Comment (BoyaWu10) [inline deploy/default-azuredeploy.json:42]: Does thee `latest` tag refer to the latest commit in master branch?

Comment (ACMoretxj) [reply deploy/default-azuredeploy.json:42]: Yes.

Comment (BoyaWu10) [reply deploy/default-azuredeploy.json:42]: I think we'd better use a stable version as default? It seems a bit risky to fetch latest commit from master as default.

Comment (BoyaWu10) [reply deploy/default-azuredeploy.json:42]: Perhaps we could release another v2.0.1 or v2.1.0?

Comment (ACMoretxj) [reply deploy/default-azuredeploy.json:42]: Sure, I believe a straightforward tag is more appropriate.

**Review by BoyaWu10 — APPROVED** [review]

**Pull request merged**